osv/ecosystems/bioconductor_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Bioconductor ecosystem helper tests."""
+
+import unittest
+
+from .. import ecosystems
+
+
+class BioconductorEcosystemTest(unittest.TestCase):
+  """Bioconductor ecosystem helper tests."""
+
+  def test_next_version(self):
+    """Test next_version."""
+    ecosystem = ecosystems.get('Bioconductor')
+    self.assertEqual('1.18.0', ecosystem.next_version('a4', '1.16.0'))
```

Comment:
@tylfin this is starting to fail, but I'm not seeing why?

```
$ GET https://packagemanager.posit.co/__api__/repos/4/packages/a4?bioc_version=1.16.0
{"code":2,"error":"No sources found for Bioconductor 1.16.0","payload":null}
$ GET https://packagemanager.posit.co/__api__/repos/4/packages/a4?bioc_version=1.18.0
{"code":2,"error":"No sources found for Bioconductor 1.18.0","payload":null}
```

Yet https://bioconductor.org/packages/release/bioc/html/a4.html suggests the package still exists?

Also:

```
$ GET https://packagemanager.posit.co/__api__/repos/4/packages/a4?bioc_version=1.50.1
{"code":2,"error":"No sources found for Bioconductor 1.50.1","payload":null}
$ GET https://packagemanager.posit.co/__api__/repos/4/packages/a4?bioc_version=1.50.0
{"code":2,"error":"No sources found for Bioconductor 1.50.0","payload":null}
```

Reply:
Sorry, I was driving that wrong, here's what's happening:

```
$ GET https://packagemanager.posit.co/__api__/repos/4/packages/a4?bioc_version=3.19
{"code":4,"error":"Package 'a4' not found","payload":null}
```

It seems to work with all of the earlier versions than 3.19, based on a spot check with the bounds of `bioc_versions`